.gitignore

```diff
@@ -47,20 +47,18 @@ PPSSPPControls.dat
 # Qt Linguist files
 *.qm
 
-#Android stuff
+# Gradle/Android Studio
+.gradle
 .idea
-android/.idea
+*.iml
+build
+build.ios
 
 Logs
 Memstick
 memstick
 Cheats
 
-bin
-gen
-libs
-obj
-build*/
```

Comment:
We've historically been telling people to create dirs like `build-ios` and `build-mac` on the root, probably better to keep this one?  Could prefix it with `/`.

-[Unknown]